develop.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [[ ! -d "docker" ]]; then
+    echo "This script must be run from the top level directory of the critiquebrainz-server source."
+    exit -1
+fi
+
+# invoke docker, but ungrep noisy influx log messages that I can't turn off
+docker-compose -f docker/docker-compose.dev.yml build && docker-compose -f docker/docker-compose.dev.yml up
```

Comment:
we should have a `-p` flag here to call the docker-compose project `critiquebrainz` instead of the default in this case which will be `docker` (the name of the directory that the docker compose file is in)